### DISCUSSION TITLE
In-place Upgrade.

### DISCUSSION THREAD

**Discussion by BrushAway** [kanboard/kanboard#5597] [Installation & Configuration]
=============================================
In-place Upgrade.
=============================================

Ticket Type: Request.
Reporter: BrushAway.
Impacted Users: Users with existing installations.
Priority: Low.
Product(s) Affected: Kanboard 1.2.35.
Deadline: N/A.
Time / Effort Estimate: N/A.

=============================================
Description
=============================================

Good afternoon all,

I have Kanboard 1.2.35 installed on my system and I was wondering if it's possible to perform an in-place upgrade on my current version?

If this is possible, can you describe how this might be done or provide a link to a place I can read about this?

Many thanks in advance for this wonderful product.

=============================================
End
=============================================

**Comment by rautamiekka** [comment]
Simply install the new version over the old one. The new one doesn't come with a preconffed config nor database, which would be the only obstacles if they existed, which is exactly the reason they don't.

**Comment by linkp** [comment]
I use a symlink from `kanboard` to `kanboard-X.XX.XX`

To upgrade, after backing up the database, I place the newly unzipped kanboard folder in the same parent folder as the existing and rsync the config file and the data and plugins folders. Once that is done, I update the symlink.

**Comment by BrushAway** [comment]
Thanks all for the feedback. I appreciate your time.
